packages/relay/tests/lib/errors/JsonRpcError.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { expect } from 'chai';
```

Comment:
Please add the license header to the top here for OSS compliance.
You'll see it in all the other `.ts` files and should be 2023

Reply:
ah, I used `packages/relay/tests/lib/formatters.spec.ts` as a reference, and it was missing a header ... assumed that it wasn't needed for test files - will add one to both now

Reply:
Thanks